app/views/alchemy/admin/resources/_filter_bar.html.erb

```diff
@@ -0,0 +1,31 @@
+<% params_to_keep = [:tagged_with, :q, :content_id, :options] %>
+
+<div id="filter_bar">
+  <h2><%= Alchemy.t('Filter') %></h2>
+  <%= select_tag(
+    'resource_filter',
+    options_for_select(
+      resource_filter_select, params[:filter]
+    ),
+    include_blank: Alchemy.t(:all, scope: "admin.#{resource_name}.filter"),
```

Comment:
Ah, it's already scoped into `alchemy`, but we should still scope it into `alchemy.resources.yada`